data_management.py

```diff
@@ -1,6 +1,24 @@
 import pandas as pd
 from datetime import datetime
 
+def del_transaction(df):
+    # data = pd.read_csv("sampledata.csv")
+    print(df)
+    while True:
+        delete = input("Enter the index of the transaction to delete: ")
+        delete = int(delete)
+        index_data = list(df.index)
+        if delete in index_data:
+            data = df.drop(delete)
```

Comment:
The valuable `data` is never used.
It should be `df`

```suggestion
            df = df.drop(delete)
```